harness/determined/cli/trial.py

```diff
@@ -120,6 +125,71 @@ def trial_logs(args: Namespace) -> None:
         timestamp_after=args.timestamp_after,
     )
 
+@authentication.required
+def output_logs(args: Namespace): 
+    if args.output_dir is None: 
+        raise Exception('Please provide the output directory you would like to direct the logs to.')
+    
+    output_dir_tar = tarfile.open(f'{args.output_dir}.tar.gz', 'w:gz')
+
+    trial_logs_file = write_trial_logs(args, args.trial_id, output_dir_tar)
+    output_dir_tar.add(f'{trial_logs_file}.json')
+    os.remove(f'{trial_logs_file}.json')
```

Comment:
It seems strange to me that half of the filename is defined inside a function a step down, and half is defined outside. 
```
stack: output_logs -> write_trial_logs -> create_json_file_in_dir
   "{filename}.json"         ^ filename            ^ "{filename}.json"
```

I feel like the extension should just be added in the `write_*` functions so there isn't an agreement between the other functions that they need to add `.json`.